parsing/intermediate/dialects/dialect1/config.fmpp

```diff
@@ -66,6 +66,11 @@ data: {
       "org.apache.calcite.sql.SqlCreateJoinIndex",
       "org.apache.calcite.sql.SqlCreateMacro",
       "org.apache.calcite.sql.SqlCreateTableDialect1",
+      "org.apache.calcite.sql.SqlCreateProcedure",
```

Comment:
nit: adjust the list of import so it follows alphabetical order

Reply:
Updated